Cargo.toml

```diff
@@ -42,9 +42,9 @@ xattr = "1.3.1"
 tempfile = "3.10.1"
 rand = { version = "0.8.5", features = ["small_rng"] }
 bytesize = "1.3.0"
-chrono = { version = "0.4.38", default-features = false, features = [
-  "clock",
-] }
+chrono = { version = "0.4.38", default-features = false, features = ["clock"] }
+walkdir = "2.5.0"
+once_cell = "1.19.0"
```

Comment:
I think this dependency is unnecessary. Some of its functionality has been added to the standard library in Rust 1.70, including the [`get_or_init`](https://doc.rust-lang.org/std/cell/struct.OnceCell.html#method.get_or_init) function you use.